controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -506,6 +512,10 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
         return false;
     }
 
+    if (database.setting_certification_mode()) {
+        return true;
```

Comment:
I think it's better to enclose the trigger itself in the condition rather than returning here. We can return because the channel selection trigger *happens* to be last thing, but if we ever add something after it, it's not so obvious that it's gated by the certification mode.

Also, I don't understand why the chennel preference query should not be sent in certification mode. The "why" should be part of the commit message. I *guess* it's because the test specification doesn't talk about the channel preference query, and conversely it does have an explicit CAPI command to trigger channel preference query. However, there is nothing that says the DUT should not send any messages that are not specified in the test plan. So I don't think this is needed at all.

Reply:
Enclosed it.
Regarding the rest of your comment - I was explained in the meeting.